src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -414,8 +442,8 @@ void Instance::HandleConnectNetwork(HandlerContext & ctx, const Commands::Connec
 
     mConnectingNetworkIDLen = static_cast<uint8_t>(req.networkID.size());
     memcpy(mConnectingNetworkID, req.networkID.data(), mConnectingNetworkIDLen);
-
-    mAsyncCommandHandle = CommandHandler::Handle(&ctx.mCommandHandler);
+    mAsyncCommandHandle      = CommandHandler::Handle(&ctx.mCommandHandler);
+    mLastOperationBreadcrumb = req.breadcrumb;
```

Comment:
So what happens if we get a ScanNetworks, then before that completes a ConnectNetwork, then the ConnectNetwork finishes first, then the ScanNetworks?  What should happen with the breadcrumb in that case per spec?

For that matter in that case, will we send the wrong sort of data to the wrong command handler, since we'll have clobbered mAsyncCommandHandle?

Reply:
This won't happen since we don't allow two background commands.

The client would receive Busy status code for the "ConnectNetwork" command.

Reply:
We have rejected the command very early:

```
void Instance::InvokeCommand(HandlerContext & ctxt)
{
    if (mAsyncCommandHandle.Get() != nullptr)
    {
        // We have a command processing in the backend, reject all incoming commands.
        ctxt.mCommandHandler.AddStatus(ctxt.mRequestPath, Protocols::InteractionModel::Status::Busy);
        ctxt.SetCommandHandled();
        return;
    }
```

Then all command handlers don't have to care about the parallel command issue.